typings/internalBinding/modules.d.ts

```diff
@@ -9,7 +9,7 @@ export type PackageConfig = {
 export type DeserializedPackageConfig = {
   data: PackageConfig,
   exists: boolean,
-  path: URL['pathname'],
+  path: string,
```

Comment:
Why? It is a URL pathname

Reply:
No, it’s a path (notable differences include `%`, `?`, and `#` do not need escaping, and on Windows, it uses `\` instead of `/`)